src/ts/scheduler.ts

```diff
@@ -74,17 +74,33 @@ document.addEventListener('DOMContentLoaded', () => {
     selectedDate = new Date(decodeURIComponent(start)).valueOf();
   }
 
+  // if no bookable items, return
+  const alertEl = document.querySelector('div[role="status"].alert-warning');
```

Comment:
make it more elegant by removing this part, rely on the presence of `#scheduler`